build-ps/debian/rules_zenfs

```diff
@@ -0,0 +1,218 @@
+DEB_BUILD_GNU_TYPE ?= $(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
+DEB_HOST_GNU_TYPE  ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
+
+DEB_SOURCE_PACKAGE ?= $(strip $(shell egrep '^Source: ' debian/control | cut -f 2 -d ':'))
+DEB_VERSION ?= $(shell dpkg-parsechangelog | egrep '^Version:' | cut -f 2 -d ' ')
+DEB_NOEPOCH_VERSION ?= $(shell echo $(DEB_VERSION) | cut -d: -f2-)
+DEB_UPSTREAM_VERSION ?= $(shell echo $(DEB_NOEPOCH_VERSION) | sed 's/-[^-]*$$//')
+DEB_UPSTREAM_VERSION_MAJOR_MINOR := $(shell echo $(DEB_UPSTREAM_VERSION) | sed -r -n 's/^([0-9]+\.[0-9]+).*/\1/p')
+DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
+
+TOKUDB_OPTS_DEFAULT ?= -DWITH_VALGRIND=OFF -DUSE_VALGRIND=OFF -DDEBUG_EXTNAME=OFF -DBUILD_TESTING=OFF -DUSE_GTAGS=OFF -DUSE_CTAGS=OFF -DUSE_ETAGS=OFF -DUSE_CSCOPE=OFF -DTOKUDB_BACKUP_PLUGIN_VERSION=$(TOKUDB_BACKUP_VERSION)
+TOKUDB_OPTS_RELEASE ?= $(TOKUDB_OPTS_DEFAULT) -DTOKU_DEBUG_PARANOID=OFF
+TOKUDB_OPTS_DEBUG ?= $(TOKUDB_OPTS_DEFAULT) -DTOKU_DEBUG_PARANOID=ON
+
+EXPORTED_SOURCE_TARBALL = debian/percona-server-source.tar.gz
+
+DISTRIBUTION = $(shell lsb_release -i -s)
+DISTRELEASE = $(shell lsb_release -c -s)
+
+MYSQL_SRC = $(shell pwd)
+
+builddir = builddir
+builddebug = debug
+
+override_dh_auto_configure:
+	@echo "RULES.$@"
+
+ifeq ($(SKIP_DEBUG_BINARY),)
+	( test -d $(builddebug) || mkdir $(builddebug) ) && cd $(builddebug) && \
+	    cmake -DBUILD_CONFIG=mysql_release \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DINSTALL_DOCDIR=share/mysql/docs \
+		-DINSTALL_DOCREADMEDIR=share/mysql \
+		-DINSTALL_INCLUDEDIR=include/mysql \
+		-DINSTALL_INFODIR=share/mysql/docs \
+		-DINSTALL_LIBDIR=lib/$(DEB_HOST_MULTIARCH) \
+		-DINSTALL_MANDIR=share/man \
+		-DINSTALL_MYSQLSHAREDIR=share/mysql \
+		-DINSTALL_MYSQLTESTDIR=lib/mysql-test \
+		-DINSTALL_PLUGINDIR=lib/mysql/plugin/debug \
+		-DINSTALL_SBINDIR=sbin \
+		-DINSTALL_SCRIPTDIR=bin \
+		-DINSTALL_SUPPORTFILESDIR=share/mysql \
+		-DSYSCONFDIR=/etc/mysql \
+		-DMYSQL_UNIX_ADDR=/var/run/mysqld/mysqld.sock \
+		-DCMAKE_BUILD_TYPE=Debug \
+		-DCOMPILATION_COMMENT=$(COMPILATION_COMMENT_DEBUG) \
+		-DROCKSDB_PLUGINS=zenfs \
+		-DSYSTEM_TYPE="debian-linux-gnu" \
+		-DINSTALL_LAYOUT=DEB \
+		-DWITH_INNODB_MEMCACHED=ON \
+		-DWITH_MECAB=system \
+		-DWITH_ARCHIVE_STORAGE_ENGINE=ON \
+		-DWITH_BLACKHOLE_STORAGE_ENGINE=ON \
+		-DWITH_FEDERATED_STORAGE_ENGINE=ON \
+		-DWITH_PAM=ON \
+		-DWITH_ROCKSDB=ON \
+		-DROCKSDB_DISABLE_AVX2=1 \
+		-DROCKSDB_DISABLE_MARCH_NATIVE=1 \
+		-DFORCE_INSOURCE_BUILD=1 \
+		-DDOWNLOAD_BOOST=1 \
+		-DWITH_BOOST=libboost \
+		-DWITH_SYSTEM_LIBS=ON \
+                -DWITH_PROTOBUF=bundled \
+                -DWITH_RAPIDJSON=bundled \
+                -DWITH_ICU=bundled \
+                -DWITH_ZSTD=bundled \
+		-DWITH_ZLIB=bundled \
+                -DWITH_LZ4=bundled \
+                -DWITH_EDITLINE=bundled \
+                -DWITH_LIBEVENT=bundled \
+                -DWITH_NUMA=ON \
+                -DWITH_LDAP=system \
```

Comment:
indentation

Reply:
Please compare with rules, you will see that there's no indentation problem, sent screenshot into slack